scout/server/blueprints/cases/templates/cases/cases.html

```diff
@@ -95,55 +96,31 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
 {% endblock %}
 
 {% macro search_form() %}
-  <form action="{{ url_for('cases.cases', institute_id=institute._id) }}" method="GET" accept-charset="utf-8">
-    <div class="row">
-      <div class="col-md-4 col-sm-4">
-        <div class="input-group">
-          <span class="input-group-addon">
-            <span class="glyphicon glyphicon-search"></span>
-          </span>
-          <input type="search" class="form-control" value="{{ query if query }}" name="query" placeholder="search cases"
-          data-toggle="tooltip" data-placement="right" data-html="true" title="
-            <div class='text-left'>
```

Comment:
Handled by the model WTF form. I don't think we need this long tooltip any more because with the new select and custom placeholders of the input field it is clear enough